contracts/toCKB-typescript/src/utils/config.rs

```diff
@@ -14,3 +14,4 @@ pub const SINCE_TYPE_TIMESTAMP: u64 = 0x4000_0000_0000_0000;
 
 // 24 * 3600 means 1 day, the unit is second
 pub const SINCE_SIGNER_TIMEOUT: u64 = LOCK_TYPE_FLAG | SINCE_TYPE_TIMESTAMP | 24 * 3600;
+pub const SINCE_AT_TERM_REDEEM: u64 = LOCK_TYPE_FLAG | 100;
```

Comment:
Unified use timestamp since type:
```suggestion
// 180 days
pub const SINCE_AT_TERM_REDEEM: u64 = LOCK_TYPE_FLAG |  SINCE_TYPE_TIMESTAMP | 180 * 24 * 3600;
```